Cargo.toml

```diff
@@ -331,13 +331,15 @@ cc = { version = "1.0.37", default-features = false }
 
 [features]
 # These features are documented in the top-level module's documentation.
-default = ["alloc", "dev_urandom_fallback"]
+default = ["alloc", "dev_urandom_fallback", "nasm"]
 alloc = []
 dev_urandom_fallback = ["lazy_static"]
 internal_benches = []
+nasm = []
```

Comment:
There's no need to add an option to use `nasm`. We should just require `nasm` unconditionally instead of `yasm`.